Resources/Prototypes/_Goobstation/tags.yml

```
@@ -175,6 +175,30 @@
+- type: Tag
+  id: ModsuitBoots
+
+- type: Tag
+  id: ModsuitChestplate
+
+- type: Tag
+  id: ModsuitCore
+
+- type: Tag
+  id: ModsuitPlatingExternal
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)